machine-runner/src/design/state.ts

```diff
@@ -145,14 +148,15 @@ export type StateMechanism<
     >,
     CommandArgs extends unknown[],
   >(
-    name: CommandName,
+    name: CommandName extends `_${string}` ? never : CommandName,
```

Comment:
what the … wow

Reply:
Wow indeed. I just realized that this is possible.